sentry/index.ts

```diff
@@ -6,12 +6,31 @@ const DSN =
   "https://1d8b45cffaf0d833276a6ad1ae7d726d@o447951.ingest.sentry.io/4505912155701248";
 const PKG_NAME = "@sentry/astro";
 interface SentryOptions {
+  /**
+   * Path to a `sentry.client.config.ts` file that contains a `Sentry.init` call.
+   * This is optional, and if not provided, a default `Sentry.init` call will made.
+   * Use this, if you want to customize your Sentry SDK setup.
+   */
+  clientInitPath?: string;
+
+  /**
+   * Path to a `sentry.client.config.ts` file that contains a `Sentry.init` call.
+   * This is optional, and if not provided, a default `Sentry.init` call will made.
+   * Use this, if you want to customize your Sentry SDK setup.
+   */
+  serverInitPath?: string;
+
```

Comment:
I guess we could also just look up the files by default in the root dir and let folks specify these options if they want to put it somewhere else 🤔